try-it.md

```diff
@@ -32,10 +32,11 @@ permalink: /try-it.html
 
 ### 1.1. Prerequisites
 
-- System with CentOS 8 or Ubuntu 18.04
+- System with CentOS 8 or Ubuntu 20.04
 - Bare metal preferred, as we will be creating VMs to emulate bare metal hosts
 - Run as a user with passwordless sudo access
-- Minimum resource requirements for the host machine: 4C CPUs, 16 GB RAM memory.
+- Minimum resource requirements for the host machine: 4C CPUs, 16 GB RAM memory
+- Required dependencies: make, kubelet, kubeadm, kubectl, go (version 1.17.5)
```

Comment:
I would suggest not to include these as dependencies since most of them will be installed by m3-dev-env scripts